packages/transport-native/package.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "@trezor/transport-native",
+    "version": "0.1.0",
+    "description": "Trezor device transport layer for React Native",
+    "main": "lib/commonjs/index",
+    "module": "lib/module/index",
+    "types": "lib/typescript/index.d.ts",
+    "react-native": "src/index",
+    "source": "src/index",
+    "files": [
+        "src",
+        "lib",
+        "android",
+        "ios",
+        "cpp",
+        "trezor-transport-native.podspec",
+        "!lib/typescript/example",
+        "!android/build",
+        "!ios/build",
+        "!**/__tests__",
+        "!**/__fixtures__",
+        "!**/__mocks__"
+    ],
+    "scripts": {
+        "test": "jest",
+        "typescript": "tsc --noEmit",
```

Comment:
please add `type-check` script for consistency across packages and to be able to use `yarn lerna run type-check` defined here https://github.com/trezor/trezor-suite/blob/develop/package.json#L31

Reply:
added